client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference.factory.js

```diff
@@ -106,14 +106,6 @@ angular.module("managerApp").factory("TelephonyGroupNumberConference", function
             featureSettings = {};
         }
 
-        if (featureSettings.announceFilename) {
-            promise.announceFilenameLabel = OvhApiMe.Document().v6().get({
```

Comment:
Please refer to [L102](https://github.com/ovh-ux/ovh-manager-telecom/pull/709/files#diff-0c89b37136efc0e565f2e98211687bc6L102) you have to remove: `var promise = {};`

And you also have to remove: `return $q.all(promise).then(function (result) {…`. Just do the `_.assign` en return `self.`